assets/js/custom/UserNotifications.js

```diff
@@ -0,0 +1,152 @@
+/* eslint-env jquery */
+/* global Swal */
+/* global FetchNotifications */
+
+// eslint-disable-next-line no-unused-vars
+function UserNotifications (markAsReadUrl, countNotificationsUrl, somethingWentWrongError,
+  notificationsClearError) {
+  const self = this
+  let all = true
+  let follower = false
+  let comment = false
+  let reactions = false
+  let remixes = false
+  self.markAsReadUrl = markAsReadUrl
+  self.countNotificationsUrl = countNotificationsUrl
+  self.somethingWentWrongError = somethingWentWrongError
+  self.notificationClearError = notificationsClearError
+
+  $(document).on('click', '#all-notif', function () {
+    if (all === false) {
+      $(this).css('background-color', '#acacac')
+      $(this).css('color', 'rgba(0,0,0,0.87)')
+      all = true
+      document.getElementById('follow-notif').style.backgroundColor = '#e0e0e0'
+      document.getElementById('follow-notif').style.color = 'rgba(0,0,0,0.87)'
+      document.getElementById('comment-notif').style.backgroundColor = '#e0e0e0'
+      document.getElementById('comment-notif').style.color = 'rgba(0,0,0,0.87)'
+      document.getElementById('reaction-notif').style.backgroundColor = '#e0e0e0'
+      document.getElementById('reaction-notif').style.color = 'rgba(0,0,0,0.87)'
+      document.getElementById('remix-notif').style.backgroundColor = '#e0e0e0'
+      document.getElementById('remix-notif').style.color = 'rgba(0,0,0,0.87)'
+      follower = false
```

Comment:
create a function to set all to false and then set the one needed to true. (move up before line 23)
Your approach has a high change that one click listener will not be updated with the new category in the future